### PR TITLE
chore(ntb): Validate SDNTB-49 Show all available languages

### PR DESCRIPTION
**Pull request by ioanpocol**

### CODE REVIEW

File: client/package.json [hunk 1]
```diff
@@ -2,7 +2,7 @@
   "name": "superdesk-client",
   "license": "GPL-3.0",
   "dependencies": {
-    "superdesk-core": "superdesk/superdesk-client-core#b3988c8c0b9aeb08896d03ca489588529a41aa78",
+    "superdesk-core": "vladnicoara/superdesk-client-core#48346a37115f2e38e4325ed76785136cdbccd862",
```
Comment (ahilles107) [inline client/package.json]: why you change dependency to you fork?

Comment (ioanpocol) [inline client/package.json]: To validate if the PR from core client pass all e2e tests. This PR is just for validation it will not be merged.

Comment (ahilles107) [inline client/package.json]: ohh... creating pr's like that will make a lot of noise and garbage in repository. Can't you configure travis to run jobs on your forks/branches?

Comment (naomiaro) [inline client/package.json]: yeah, this stuff can start to get merged by accident too....

Comment (ioanpocol) [inline client/package.json]: For all PR in client core we need to validate against e2e tests.
Petr will move the running of e2e tests on client core so this is a temporary solution. 
I set with wip label all those PR.
@naomiaro If you think that a improvement is still needed you can try to do it as an example with your PR:
https://github.com/superdesk/superdesk-client-core/pull/136

Comment (ahilles107) [inline client/package.json]: But i think that you still can add your fork (https://github.com/ioanpocol/superdesk) to travis? Then travis will run tests against commits/branches there and that PR will be not needed.

**Comment by naomiaro** [comment]
So we are merging stuff pointing to our personal repositories?

**Comment by ahilles107** [comment]
Looks like this PR was closed automatically by merging branch (but with correct repository name in deps) But agree it looks bad.

**Pull request merged**